app/models/responses/PropertyAnnualSubmission.scala

```diff
@@ -121,6 +122,45 @@ object PropertyAnnualSubmission {
     resultWithSbas
   }
 
+  def fromRentalsAndRentARoomAbout(
```

Comment:
Let's keep an eye on this file, as it’s growing. We also need to consider how much of it we can reuse.

Reply:
yes.